Packs/MongoDB/Integrations/MongoDB/MongoDB.py

```diff
@@ -234,6 +256,73 @@ def convert_object_id_to_str(entries: List[ObjectId]) -> List[str]:
     return list(map(str, entries))
 
 
+def handle_bulk_update_string_query_arguments(
+        argument: Tuple[str, str],
+        filter_valid_input_example: str, update_valid_input_example: str) -> List[dict]:
+    """Handles the case where the filter or update arguments are strings when using mongodb-bulk-update command.
+        (the other case is a list of dictionaries from context)
+
+    Args:
+        argument (Tuple[str, str]): a tuple containing the argument name and value.
```

Comment:
filter_valid_input_example and update_valid_input_example are missing in the docstring.

Reply:
Done.